nestkernel/nestmodule.cpp

```diff
@@ -2155,24 +2153,24 @@ NestModule::GetPosition_gFunction::execute( SLIInterpreter* i ) const
 }
 
 /** @BeginDocumentation
-  Name: topology::Displacement - compute displacement vector
+  Name: nest::Displacement - compute displacement vector
 
   Synopsis: layer from_node_id to_node_id Displacement -> [double vector]
             layer from_pos to_node_id Displacement -> [double vector]
 
   Parameters:
   layer       - NodeCollection for layer
-  from_node_id    - int, node_id of node in a topology layer
+  from_node_id    - int, node_id of node in a spatial layer
   from_pos    - double vector, position in layer
```

Comment:
I know this doesn't have anything to do with this PR, but could you add some space so all the `-` are aligned?